CIScripts/Build.ps1

```diff
@@ -41,48 +40,39 @@ foreach ($Directory in $Directories) {
     }
 }
 
-$ComponentsToBuild = Get-ComponentsToBuild
-
 try {
-    if ("DockerDriver" -In $ComponentsToBuild) {
```

Comment:
If you're removing ComponentsToBuild, please also remove them in `BuildStage.ps1`